### PR TITLE
remove the unnecessarily settings

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

**Comment by saeedmb8** [comment]
@Mitso quick review please

File: gem/settings/base.py [hunk 1]
```diff
@@ -127,7 +127,7 @@
 
 # We have multiple layouts: use `base`, `malawi` or `springster`
 # to switch between them.
-SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'springster')
+SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'base')
```
Comment (Mitso) [inline gem/settings/base.py]: @saeedmb8 is changing this to base intentional?

**Comment by Mitso** [comment]
@saeedmb8 otherwise a 👍

File: gem/settings/base.py (hunk 1 above)
Comment (saeedmb8) [inline gem/settings/base.py]: @Mitso yes, for now on QA and prod the base site layout is gonna be the `base` and those that we want to be `springster` we can change the value of `SITE_LAYOUT_BASE` on MC

**Pull request merged**